src/product-and-services/index.md

```diff
@@ -4,20 +4,24 @@ title: Products and Services
 ---
 ## In Active Development
 
-### [Notabli](https://www.notabli.com)
+### [Convene](https://www.zinc.coop/convene)
 
-Notabli helps parents safely share their children's childhood moments with their extended family. We spend four to eight days a month updating [Notabli](https://www.notabli.com).
+[Convene](https://www.zinc.coop/convene)(Convene creates human connection through safe, accessible and reliable video conferencing.
```

Comment:
`/www.zinc.coop/convene)(Convene creates human`

It looks like there's an extra`(` here in `zinc.coop/convene)(Convene`

Reply:
Good catch. Fixed in https://github.com/zinc-collective/www.zinc.coop/pull/51